float32 relevance
int32 indexCategory
---
